atomistics/referencedata/__init__.py

```
@@ -9,7 +11,11 @@
     from atomistics.referencedata.wolframalpha import (
         get_chemical_information as get_chemical_information_from_wolframalpha,
     )
-except ImportError:
+except ImportError as e:
+    warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)